packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js

```diff
@@ -14,10 +12,23 @@ storiesOf('Patterns (Sub-Patterns)|ContentGroup', module)
     },
   })
   .add('Default', () => {
-    const copy = text(
-      'Component title(required):',
+    const heading = text(
+      'Component heading(required):',
       'Lorem ipsum dolor sit amet.'
     );
+    const renderCTA = boolean('render CTA', true);
+    const cta = {
+      title: renderCTA && text('CTA heading', 'Lorem ipsum dolor sit amet'),
+      href: renderCTA && text('href', 'https://ibm.com'),
+    };
 
-    return <ContentGroup heading={copy} />;
+    return (
+      <div className="bx--grid ">
+        <div className="bx--row">
+          <div className="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
+            <ContentGroup heading={heading} {...(renderCTA && { ...cta })} />
```

Comment:
`{...(renderCTA && { ...cta })}` looks weird

can we do `{renderCTA &&  ...cta}` instead?

Reply:
Okay.